.github/workflows/pytest.yml

```
@@ -0,0 +1,29 @@
+name: Pytest
+
+on:
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)